### PR TITLE
Document the light client interface

### PR DESCRIPTION
**Pull request by rtyshko**
AVA-327

### CODE REVIEW

File: PUBLIC_DOCS.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+## Endpoints
+
+Return the Mode of the Light Client
+
+```bash
+curl -s localhost:7000/v1/mode
+```
+
+Result:
```
Comment (prabal-banerjee) [inline PUBLIC_DOCS.md:9]: Can we call these results as "Sample Result"?

File: PUBLIC_DOCS.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+Return the status of a latest block 
+
+```bash
+curl -s localhost:7000/v1/status
+```
+
+Result:
+
+```json
+{"block_num":2,"confidence":93.75,"app_id":0}%
```
Comment (prabal-banerjee) [inline PUBLIC_DOCS.md:68]: Seems like there is a trailing `%`. Kindly recheck.

File: PUBLIC_DOCS.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+    "confidence": 99.90234375,
+    "serialisedConfidence": "958776730446"
+}
+```
+
+>  `serialisedConfidence` is calculated as: 
+> `blockNumber << 32 | int32(confidence * 10 ** 7)`, where confidence is represented out of 10 ** 9.
+
+
+Given a block number (as _(hexa-)_ decimal number), return the content of the data if specified in config in hex string format
```
Comment (prabal-banerjee) [inline PUBLIC_DOCS.md:47]: The sentence might be a bit confusing to a new user. Can we rephrase?

Comment (rtyshko) [reply PUBLIC_DOCS.md]: Fixed

File: PUBLIC_DOCS.md (hunk 2 above)
Comment (rtyshko) [reply PUBLIC_DOCS.md]: Fixed

**Pull request closed without merging**